package.json

```diff
@@ -25,43 +25,43 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "nest build",
-    "prepare": "yarn run build",
+    "prepack": "yarn run build",
```

Comment:
why is this change? 
cc @MiffyLiye

Reply:
similar to https://github.com/Wiredcraft/nestjs-ottoman/pull/3